docs/go/quickstart/gorillamux-sql.md

```diff
@@ -0,0 +1,113 @@
+---
+id: gorillamux-sql
+title: Sample Application with Gorilla/Mux-SQL
+sidebar_label: Gorilla/Mux-SQL Sample
+tags:
+  - hello-world
+  - go
+  - sdk
+  - tutorial
+---
+
+# URL Shortener Sample App
```

Comment:
The entire doc for this app would be a product catalogue app. You can try running the sample app! Do let me know if you face any issues!